src/api/helpers/siteConvertingHelpers.ts

```diff
@@ -26,15 +26,17 @@ export const convertSiteToAvailableParticipantDTO = (
   return {
     name: matchedParticipant ? matchedParticipant.name : site.name,
     siteId: site.id,
-    types:
-      matchedParticipant && matchedParticipant.types
```

Comment:
I agree to use site.clientTypes as a source of truth. Just wondering currently some of the participants in integ are just using random site ids, Are we going to fix those data? otherwise, their type will not matches the type they have in portal

Reply:
If the type in the portal doesn't match admin, then the type in admin should win for deciding who can be shared with. Recommendations it doesn't matter so much, because the participant can still decide what to do.